src/routes/(auth)/login/+page.svelte

```diff
@@ -1,12 +1,23 @@
 <script>
 	import { authHandlers } from '$lib/handlers/auth';
 	import { goto } from '$app/navigation';
+	import Spinner from '$lib/components/Spinner.svelte';
+
 	let email = '';
 	let password = '';
+	let sendingRequest = false;
+	let loginError;
 
 	const handleSubmit = async () => {
-		await authHandlers.login({ email, password });
-		goto('/dashboard');
+		try {
+			sendingRequest = true;
+			await authHandlers.login({ email, password });
+			goto('/dashboard');
+		} catch (err) {
+			console.log(err);
+			sendingRequest = false;
+			// To do: Handle error here
+		}
```

Comment:
Maybe a `finally` where we set `sendingRequest = false` like in the reser-password page?

Reply:
I intentionally set this one without a finally block because if the try block succeeds and the user is redirected to dashboard, finally block will never be reached. We can add a finally block though, no problem

Reply:
fair enough
I would add it for consistency, though, so that it's similar to the other one

Reply:
I agree. let's keep things consistent

Reply:
Alright, done!